### PR TITLE
feat: ask user confirmation on unpair

### LINKED ISSUE #403: Ask user why they are unpairing the desktop app
Include a tick box: "Don't ask me again"

### PR DESCRIPTION
**Pull request by OGPoyraz**
## Overview
This PR implements a confirmation dialog when the user tries to unpair while paired.

### CODE REVIEW

File: packages/app/src/app/desktop-app.ts [hunk 1]
```diff
@@ -103,11 +103,27 @@ class DesktopApp extends EventEmitter {
...
+      dialog
+        .showMessageBox({
+          type: 'info',
+          title: t('unpairMetaMaskDesktop'),
+          message: t('unpairMetaMaskDesktopDesc'),
+          buttons: [t('yes'), t('no')],
+        })
+        .then(async (value) => {
+          if (value.response === 0) {
+            this.metricsService.track(EVENT_NAMES.DESKTOP_APP_UNPAIRED, {
```
Comment (matthewwalsh0) [inline packages/app/src/app/desktop-app.ts:120]: Do we not want this inside the `disable` method so it's done at the source?

**Review by matthewwalsh0 — APPROVED** [review]

File: packages/app/src/app/desktop-app.ts (hunk 1 above)
Comment (OGPoyraz) [reply packages/app/src/app/desktop-app.ts:120]: Good question, it actually depends if we want to capture this event regardless of the answer for this confirmation. 
I will ask @vinistevam

Comment (vinistevam) [reply packages/app/src/app/desktop-app.ts:120]: Yeap makes sense to move it to `disable`.

Comment (OGPoyraz) [reply packages/app/src/app/desktop-app.ts]: done

**Review by vinistevam — APPROVED** [review]
Great PR :rocket:

**Review by cryptotavares — APPROVED** [review]

**Comment by abdulmenan10** [comment]
Hf6

**Pull request merged**